### DISCUSSION TITLE
QSelect text color

### DISCUSSION THREAD

**Discussion by denprog** [quasarframework/quasar#16528] [General - Components / Directives / etc] [answered]
How can I set a color for a selected item of the QSelect component? It's always black.
`
<q-select  v-model="locale" :options="localeOptions" @update:model-value="onLanguage();" dense borderless no-caps flat emit-value map-options options-dense style="padding-left:10px;padding-right:10px;" />
`
Code like this sets color for all the QSelect components, but I'd like for only one (or a group):
`
.q-field__native
{
    color: white !important;
}
`

**Comment by metalsadman** [comment]
```css
.q-field__native > span {
  color: whatevercoloryoulike;
}
```

**Comment by denprog** [comment, marked as answer]
This is does all the selects of this color. Found the answer via a css class:
`<q-select v-model="locale" class="language-select" :options="localeOptions" @update:model-value="onLanguage();" dense borderless no-caps flat emit-value map-options options-dense style="padding-left:10px;padding-right:10px;" />
`
`.language-select
{
    .q-field__native
    {
        color: white;
    }
}`
